docs/docs/cmd/request.md

```diff
@@ -0,0 +1,64 @@
+# request
+
+Executes the specified web request using CLI for Microsoft 365
+
+## Usage
+
+```sh
+m365 request [options]
+```
+
+## Options
+
+`-u, --url <url>`
+: The request URL. 
+
+`-m, --method [method]`
+: The HTTP request method. Accepted values are `get, post, put, patch, delete, head, options`. The default value is `get`.
+
+`-r, --resource [resource]`
+: The resource uri for which the CLI should acquire a token from AAD in order to access 
+the service.
+
+`-b, --body [body]`
+: The request body. Optionally use `@example.json` to load the body from a file. 
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+The request will be issued as bare as possible, meaning you are responsible for most request information, such as headers, method and body. There are a few exceptions: 
+
+- the command does apply compression and throttling handling as part of the request execution. 
+- The `accept` header is set and defaults to `application/json`. 
+- The `authorization` header is set. By default, the command will try to retrieve a valid token for the API you are executing a request against based on the `url` option. 
+
+If you specify the `resource` option, the CLI will try to retrieve a valid token for the resource instead of determining the resource based on the url. The value doesn't have to be a URL. It can be also a URI like `app://<guid>`.
```

Comment:
Thinking of this... I wonder if we don't have a gap for retrieving access tokens for single-tenant apps, where you need to specify the tenant ID as well 🤔

Reply:
Could you explain that some more? We're using the common endpoint by default, right? Unless you use the environment variable...